### ISSUE TITLE
pepr and RnBeads

### ISSUE THREAD

**Issue by nsheff** [epigen/RnBeads#3]
I just wanted to rekindle a conversation we started awhile ago;

it would be fantastic if RnBeads could just run on on a standard PEP-formatted project. This would give increased access to RnBeads to using a standardized way of organizing project metadata.

The details on the format itself are here: https://pepkit.github.io/
And we could use the `pepr` package to outsource all the metadata management, so it doesn't have to be done within RnBeads. see here: https://pepkit.github.io/docs/pepr/

Any interest?

**Comment by lutsik** [comment]
Worth being explored, although I am not sure I can find even a bit of time for this. BTW, is there any crossover (planned?) with the CWL ecosystem: http://www.commonwl.org ?

**Comment by nsheff** [comment]
Well, I've been thinking about it. PEP and CWL handle fundamentally different things; CWL is for building pipelines, PEP is for describing sample metadata. In theory, a CWL tool could employ PEP concepts for managing metadata. That would be great if they did that. So I would see this as a separate project that would sit at the interface of the two. Maybe I will contact some people working on some of those projects to gauge interest.

**Comment by lutsik** [comment]
Yes, you should definitely let them know. Their project lead, Michael Crusoe (https://github.com/mr-c) is usually very responsive. We recently met here in Heidelberg, so you can CC me.

**Comment by nsheff** [comment]
@demuellae  No interest at all in using PEP for RnBeads, then? I still think this is a great idea. To follow up, we've been working with CWL people and have made quite some progress on integration there.

**Comment by nsheff** [comment]
Also, I can point out we released version 2.0 of PEP last year which made it both simpler to use and more powerful, so it might be worth another look. The new docs are here: http://pep.databio.org/en/latest/

**Comment by demuellae** [comment]
Yes, still interested. Reopening. We just weren't sure, whether PEP support is still something of interest, since the conversation took place a while ago. We just cleaned out some old Git issues. Unfortunately, we have low bandwidth at the moment, though

**Comment by nsheff** [comment]
ok, great. Let's leave this open, we might try to take a look in the next couple of weeks.
